src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,161 @@
+import React, {
+  useState,
+  useRef,
+  useMemo,
+  useCallback,
+  useEffect,
+} from 'react';
+import { debounce } from 'lodash';
```

Comment:
Fix it
<img width="769" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/6fce7760-6363-48e9-9174-7627d68404c7">

Reply:
it`s been already installed by me before the previous commit 
![image](https://github.com/mate-academy/react_autocomplete/assets/103532899/1688728c-e321-49e4-92c1-0e62fe6b2e5c)